### PR TITLE
#30 Adding support for "--config" flag for CMakeBuild

### PR DESCRIPTION
**Pull request by Breakpoint21**

### CODE REVIEW

**Review by patriksvensson — APPROVED** [review]
LGTM! 👍

**Comment by patriksvensson** [comment]
Let's wait for @moongdev to have the final word about this change since he's been the active maintainer for a while now.

**Review by darlov — APPROVED** [review]

**Pull request merged**
